apps/spark/src/main/java/com/linkedin/openhouse/jobs/spark/Operations.java

```diff
@@ -217,9 +220,11 @@ public void expireSnapshots(Table table, int maxAge, String granularity, int ver
     ExpireSnapshots expireSnapshotsCommand = table.expireSnapshots().cleanExpiredFiles(false);
 
     // maxAge will always be defined
-    TimeUnit timeUnitGranularity = TimeUnit.valueOf(granularity.toUpperCase());
+    ChronoUnit timeUnitGranularity =
```

Comment:
For understanding how ChronoUnit differs from TimeUnit?

Reply:
TimeUnit only goes up to DAYS, doesn't support months/year